### PR TITLE
Update Korean localization

### PR DESCRIPTION
**Pull request by quiple**
Update the Korean localization of added and changed strings.

- Localizes strings related to plural.
- Removes deprecated strings.

Thank you!

### CODE REVIEW

File: src/locale/locales/ko/messages.po [hunk 1]
```diff
@@ -8,80 +8,80 @@ msgstr ""
...
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:241
 msgid "{0, plural, one {Unlike (# like)} other {Unlike (# likes)}}"
-msgstr ""
+msgstr "좋아요 취소 ({0}개)"
 
 #: src/components/LabelingServiceCard/index.tsx:71
 msgid "{count, plural, one {Liked by # user} other {Liked by # users}}"
-msgstr ""
+msgstr "{0}명의 사용자가 좋아함"
```
Comment (auroursa) [inline src/locale/locales/ko/messages.po:76]:
```suggestion
msgstr "{count}명의 사용자가 좋아함"
```

File: src/locale/locales/ko/messages.po [hunk 2]
```diff
@@ -8,80 +8,80 @@ msgstr ""
...
 
 #: src/screens/Deactivated.tsx:207
 msgid "{estimatedTimeHrs, plural, one {hour} other {hours}}"
-msgstr ""
+msgstr "시간"
```
Comment (auroursa) [inline src/locale/locales/ko/messages.po:80]:
```suggestion
msgstr "{estimatedTimeHrs}시간"
```

File: src/locale/locales/ko/messages.po [hunk 3]
```diff
@@ -8,80 +8,80 @@ msgstr ""
...
 
 #: src/screens/Deactivated.tsx:213
 msgid "{estimatedTimeMins, plural, one {minute} other {minutes}}"
-msgstr ""
+msgstr "분"
```
Comment (auroursa) [inline src/locale/locales/ko/messages.po:84]:
```suggestion
msgstr "{estimatedTimeMins}분"
```

File: src/locale/locales/ko/messages.po [hunk 4]
```diff
@@ -92,44 +92,31 @@ msgstr "{following} 팔로우 중"
 #: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:298
 #: src/view/screens/ProfileFeed.tsx:604
 msgid "{likeCount, plural, one {Liked by # user} other {Liked by # users}}"
-msgstr ""
+msgstr "{0}명의 사용자가 좋아함"
```
Comment (auroursa) [inline src/locale/locales/ko/messages.po:95]:
```suggestion
msgstr "{likeCount}명의 사용자가 좋아함"
```

**Review by auroursa — COMMENTED** [review]
Some of the beginnings are not 0, specific strings need to be used.

**Comment by quiple** [comment]
@auroursa Thanks!

File: src/locale/locales/ko/messages.po [hunk 5]
```diff
@@ -8,80 +8,80 @@ msgstr ""
...
 
-#: src/view/com/modals/VerifyEmail.tsx:151
+#: src/view/com/modals/VerifyEmail.tsx:150
 msgid "(no email)"
 msgstr "(이메일 없음)"
 
 #: src/view/com/notifications/FeedItem.tsx:239
 msgid "{0, plural, one {{formattedCount} other} other {{formattedCount} others}}"
-msgstr ""
+msgstr "외 {0, plural, {{formattedCount}}}명"
```
Comment (auroursa) [inline src/locale/locales/ko/messages.po:22]:
```suggestion
msgstr "외 {0, plural, other {{formattedCount}}}명"
```

**Review by auroursa — COMMENTED** [review]
Ignoring "other" here will cause compilation error

**Review by pfrazee — APPROVED** [review]
Thanks so much!!

**Pull request merged**